### DISCUSSION TITLE
Operation timed out

### DISCUSSION THREAD

**Discussion by pixelartes** [iterate-ch/cyberduck#15002] [Q&A]
I keep getting the following error message when uploading my website files and folders using Cyberduck:
"Operation timed out. The connection attempt was rejected. The server may be down, or your network may not be properly configured."

The upload gets so far before I get the message.

Any help would be appreciated.

**Comment by tatze96** [comment]
"Operation timed out" errors often occur if there are connectivity issues in the local network or the server doesn't answer fast enough on a request.

You can try to work around the error by increasing the "Timeout for opening connection" within the Cyberduck preferences _Connection_ tab. Additionally, you can increase the "Number of retries" for failed network tasks within the same Cyberduck preferences section to let Cyberduck retry the connection multiple times by itself before showing a connection error.

Please send us the logging output for further investigation if this doesn't improve the timeout errors. To do so, [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), connect to the server, recreate the error, and send us the log file afterward. If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15002 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
